app/scripts/templates/force_auth.mustache

```diff
@@ -14,7 +14,7 @@
     </div>
     <form novalidate>
       <p class="prefill">{{ email }}</p>
-
+      <input type="email" class="email hidden" value="{{ email }}" />
```

Comment:
Would it behoove us to add the `disabled` attribute to give some semantic indication that this input cannot be edited?